scipy/stats/tests/test_resampling.py

```diff
@@ -1001,20 +1001,22 @@ def statistic(x, y, axis):
 
     # -- Independent (Unpaired) Sample Tests -- #
 
-    def test_against_kstest(self):
+    @pytest.mark.parametrize('alternative', ("less", "greater", "two-sided"))
+    def test_against_kstest(self, alternative):
         np.random.seed(0)
         x = stats.norm.rvs(size=4, scale=1)
         y = stats.norm.rvs(size=5, loc=3, scale=3)
```

Comment:
This one's for you, @tupui.
```suggestion
        rng = np.random.default_rng(abs(hash("ks_2samp")))
        x = rng.normal(size=4, scale=1)
        y = rng.normal(size=5, loc=3, scale=3)
```